test/userdata/agent_configs/cpu_config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "agent": {
+    "metrics_collection_interval": 15,
+    "debug": true,
+    "logfile": ""
+  },
+  "metrics": {
+    "namespace": "UserDataTest",
+    "append_dimensions": {
+      "InstanceId": "${aws:InstanceId}"
+    },
+    "metrics_collected": {
+      "cpu": {
+        "measurement": [
+          {"name": "time_active", "rename":"cpu_time_active_renamed"},
```

Comment:
We don't need to test the full set of CPU metrics. The purpose of this test is to be a smoke test to ensure that we can install and run the agent e2e when installed and configured via user data scripts. I think we can trim down what we validate here. I think the rename is a nice touch.

Reply:
I removed all other CPU tests and kept it to `time_active` only